Update Review.java

huntermargaux opened the pull request:
Give me feedback!
@jnpruett
@ryanrebne
@ncannapruett
@bdurling3
@Briyannis
@sevandersip

huntermargaux committed:
Update Review.java

JnPruett commented:
totalSentiment() method looks great! Good to merge.

ncannapruett commented:
I'm confused to what the line
review.replaceAll("\\p{Punct}", "");
means, but hopefully it works!
But besides that, your totalSentiment() looks great!

huntermargaux committed:
totalSentiment() method

This method loops through a review, and adds up the sentiment value of all of the words within the review, based on their sentiment value in the spreadsheet document. I used two variables, an empty String to collect each word as the program loops through the review, and a double value to hold the total sentiment value; each word's value is added to this variable after the completion of the loop through said word.
